### ISSUE TITLE
add option for seperate log file "suffix" location type to replace original image extension

### ISSUE THREAD

**Issue by Hugs288** [Bionus/imgbrd-grabber#3325]
currently if i download an image with .txt suffix, the txt file will be saved as etcetc.jpg.txt, but i want it to save as just .txt.

**Comment by Bionus** [comment]
Makes sense, although note that they would overlap if you download files with the same path but a different extension 👍

**Comment by Hugs288** [comment]
yes, particularly ive realized the %num% option (when used for filenames atleast) can give the same number to an image with a different extensions. so you can have 2.jpg and 2.png end up in the same folder and its very annoying, it would be much more useful if it just game a unique number to every image regardless of extension.
